src/views/Player.vue

```diff
@@ -186,6 +187,22 @@ export default {
 
           // set the global list of time values
           this.times = res.data.times
+
+          // merge the previous session data
+          if (res.data.sessionData != null) {
+            this.answers = res.data.sessionData.answers
+            for (i=0; i < questions.length; i++){
+              this.plioQuestions[i].user_answer = this.answers[i]
+              this.plioQuestions[i].state = 
+                (this.answers[i].length == 0) ? "notshown" : "answered"
+            }
+            
+            this.journey = res.data.sessionData.journey
+            this.previousPlayerTime = (
+              (this.journey.length > 0) ? this.journey[this.journey.length - 1]['player_time'] : [])
+            this.watchTime += res.data.sessionData['watch-time']
+            this.retention = res.data.sessionData.retention
+          }
```

Comment:
If previous session data is available,
1. copy the answers into `this.answers`
2. copy the answers into the `plioQuestions` object as this is what is passed to the `PlioQuestion.vue` component
3. Also set the state as `answered` if the question was answered by the student in previous sessions
4. copy the journey
5. take the `player_time` from the very last item from `journey` array. This is where the new session's video will start playing from
6. add the watch-time to the existing watch-time
7. copy the retention array